applications/tari_dan_wallet_cli/src/command/account.rs

```diff
@@ -90,8 +96,23 @@ pub struct ClaimBurnArgs {
     /// Optional proof JSON from the L1 console wallet. If not provided, you will be prompted to enter it.
     #[clap(long, short = 'j', alias = "json")]
     proof_json: Option<serde_json::Value>,
-    #[clap(long, short = 'f')]
-    fee: Option<u64>,
+    #[clap(long, short = 'f', default_value_t = 1000)]
+    fee: u64,
+}
+
+#[derive(Debug, Args, Clone)]
+pub struct RevealFundsArgs {
+    /// The account name where the funds will be revealed
+    account_name: String,
+    /// Amount of funds to reveal
+    reveal_amount: u64,
+    /// The fee to pay for the reveal transaction
+    #[clap(long, short = 'f', default_value_t = 1000)]
+    fee: u64,
+    /// If set, the fee will be paid from the revealed funds instead of from the account resulting in less revealed
+    /// funds than requested.
+    #[clap(long)]
+    pay_from_reveal: bool,
```

Comment:
This is a difficult thing to name, suggesting this, but I don't think it's any better:
```suggestion
   use_as_fees: bool,
```

Reply:
yeah `--pay-fees-from-revealed-funds` is probably better but a bit long. Perhaps we should always just pay from revealed funds since it amounts to the same final amount and it works even when no funds are previously revealed